src/azure-cli/azure/cli/command_modules/acr/commands.py

```diff
@@ -293,3 +297,20 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
 
     with self.command_group('acr token credential', acr_token_credential_generate_util) as g:
         g.command('generate', 'acr_token_credential_generate')
+
+    with self.command_group('acr private-endpoint-connection', acr_private_endpoint_connection_util,
+                            is_preview=True) as g:
+        g.command('delete', 'delete')
+        g.command('show', 'show')
+        g.command('list', 'list_')
```

Comment:
is there any specific reason to use `list_` instead of `list`?

Reply:
`list` is a reserved keyword of python. Appending `_` is suggested by [PEP8](https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments).

Reply:
good to learn.